src/flows/pipes/Notification/Threshold.tsx

```diff
@@ -54,7 +54,6 @@ const Threshold: FC<Props> = ({readOnly}) => {
       }
 
       threshold.type = type
-      threshold.field = threshold?.field
```

Comment:
>this could just be an artifact of a refactor that wasn’t fully cleaned up.

@asalem1 I think that's exactly what happened:

https://github.com/influxdata/ui/commit/4f09cc11304fd49aba3e8319e23313f63114870e#diff-0637c8cc7347182900e160d4d2985cb2d71a508d8a8bae50977fe125dcdc178fR57